OWNERS

```diff
@@ -1,4 +1,7 @@
-approvers:
-  - haircommander
-  - rphillips
-  - saschagrunert
+filters:
+  .*:
+    reviewers:
+    - cri-o-reviewers
+    approvers:
+    - cri-o-approvers
+    - rphillips
```

Comment:
Why not put Ryan in the approvers group? :smile:

Reply:
@rphillips is not part of that group in CRI-O :shrug:

Reply:
A special snowflake, I see. :snowflake: